apps/mark-scan/daemon-utils/src/lib.rs

```diff
@@ -0,0 +1,43 @@
+use std::{
+    sync::{
+        atomic::{AtomicBool, Ordering},
+        Arc,
+    },
+    thread::sleep,
+    time::{Duration, Instant},
+};
+use vx_logging::{log, EventId, EventType};
+
+const HEARTBEAT_INTERVAL: Duration = Duration::new(5, 0);
+const NOOP_LOOP_INTERVAL: Duration = Duration::new(1, 0);
```

Comment:
Use factory functions that make the value clearer?
```suggestion
const HEARTBEAT_INTERVAL: Duration = Duration::from_secs(5);
const NOOP_LOOP_INTERVAL: Duration = Duration::from_secs(1);
```